aim/web/ui_v2/src/components/ChartPanel/ChartPanel.tsx

```diff
@@ -98,6 +98,8 @@ const ChartPanel = React.forwardRef(function ChartPanel(
             {props.data.map((lineChartData, index) => (
               <Grid
                 key={index}
```

Comment:
shouldn't we change the key logic to avoid drawing an additional first chart?

Reply:
Yes We Are, but we need to rerender LineChart when chart grouping state changes. So with the logic we wrote LineChart also rerenders when we changing style and color grouping state. I wrote TODO for this issue.